eogrow/utils/types.py

```diff
@@ -16,6 +16,8 @@
 
 BoolOrAuto = Union[Literal["auto"], bool]
 
+JsonDict = Dict[str, Any]
```

Comment:
Hmmm, we actually use the same definition of `JsonDict` also in sh-py. Here I just didn't want to import it from sh-py in case we'll change something there. The reason we name it `JsonDict` is because it is meant as a type of dictionary that could be serialized into JSON. Although, the type itself is not precise enough for that and will have to be improved in the future.

I'm really not sure about this one. For now I just wanted to have the same name as in sh-py but we can also change it.

CC @zigaLuksic

Reply:
Keeping it the same as in `sh-py` is probably a good enough reason. I do agree that we can revisit the naming at some later time.